src/gauges/FraxUnifiedFarmTemplate.sol

```diff
@@ -34,7 +34,7 @@ pragma solidity >=0.8.0;
 // (Locked, veFXS, and UniV3 portions are new)
 // https://raw.githubusercontent.com/Synthetixio/synthetix/develop/contracts/StakingRewards.sol
 
-import {console} from "forge-std/Test.sol";
+import "lib/forge-std/src/Test.sol";

```

Comment:
not big deal, but its always a good practice to import what you need to avoid collisions 👍🏻